content/github/developing-online-with-codespaces/configuring-codespaces-for-your-project.md

```diff
@@ -73,13 +72,17 @@ You can use configuration keys supported by {% data variables.product.prodname_c
 - `remoteEnv`
 - `containerUser`
 - `remoteUser`
-- `updateRemoteUserUID`
 - `mounts`
-- `workspaceMount`
-- `workspaceFolder`
+- `workspaceMount` (Not yet supported)
+- `workspaceFolder` (Not yet supported)
 - `runArgs`
 - `overrideCommand`
-- `shutdownAction`
 - `dockerComposeFile`
 
+#### Not applicable for Codespaces (no plan to support)
+
+- `devPort`
+- `updateRemoteUserUID`
+- `shutdownAction`
+
```

Comment:
Let's remove this second list as well. If we want to make it crystal clear that anything not in the existing list is unsupported, we could add this sentence to L59!

>Any keys not in the list are not supported by {% data variables.product.prodname_codespaces %}. 

```suggestion
```